.devcontainer/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.234.0/containers/typescript-node/.devcontainer/base.Dockerfile
```

Comment:
Thank you for the docker set up.
It should be in a separate PR, please, with notes in CONTRIBUTING on how to use it.

Also, could those notes say how this relates to the Docker setup in the docs directory? Does this replace the docker setup, or would both containers be needed?

Reply:
backed the change out of this PR